en/controllers/components/security.rst

```diff
@@ -169,51 +144,14 @@ want and the Security Component will enforce them on its startup::
         {
             parent::beforeFilter($event);
 
-            if ($this->request->getParam('admin')) {
-                $this->Security->requireSecure();
+            if ($this->request->getParam('prefix') === 'admin') {
+                $this->Security->setConfig('validatePost', false);
```

Comment:
most cases in admin area `validatePost` should be `true` as is by default 
could you change example ?

Reply:
Not really, I turn off SecurityComponent (actually just don't load it) for admin area :slightly_smiling_face:.

Reply:
There aren't many options left to turn off. We could remove this section entirely.

Reply:
for example for requests from other sites/server to very(payment)/... or other something
@ADmad if you was only admin yes you can remove whole security features :smile: